tutormfe/plugin.py

```diff
@@ -304,3 +305,27 @@ def _check_mfe_host(config: Config) -> None:
             f'Warning: MFE_HOST="{mfe_host}" is not a subdomain of LMS_HOST="{lms_host}". '
             "This configuration is not typically recommended and may lead to unexpected behavior."
         )
+    
+
+@tutor_hooks.Actions.CONFIG_LOADED.add()
+def _run_jobs_in_mounted_mfes(config: Config) -> None:
+    mounts = get_typed(config, "MOUNTS", list, [])
+
+    pattern = re.compile(r'frontend-app-(\w+)')
```

Comment:
We can use the REPO_PREFIX variable defined above here instead of a hardcoded string.